### ISSUE TITLE
USB Connection for Firmware change

### ISSUE THREAD

**Issue by anlyoung** [ROBOTIS-GIT/turtlebot3#186]
If I would try to play with the firmware - modify and upload via Arduino IDE in the host computer.. shall I connect the usb cable to the OpenCR directly, not to the Joule board? That's what I would assume but hope to confirm.

**Comment by robotpilot** [comment]
Yes. You can upload it by connecting directly to OpenCR with your PC(not Joule board) after programming(modify, re-build) on your PC.

**Comment by anlyoung** [comment]
Thank you for the prompt reply.

There is a microUSB port in the Joule board. It has not been used in the manual, correct?

-- Young Park

________________________________
